entries/t/trendmicro.com.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Trend Micro": {
+    "domain": "trendmicro.com",
+    "url": "https://www.trendmicro.com",
+    "tfa": [
+      "sms",
+      "email",
+      "totp"
```

Comment:
I don't see any mention of which 2FA methods Trend Micro supports. Could you link to a page that shows that, or if no such page can be found, add a screenshot where the supported methods are displayed?

Reply:
<img width="878" alt="Screenshot 2023-10-25 at 11 13 07 AM" src="https://github.com/2factorauth/twofactorauth/assets/21183744/d369aa53-b27e-42e2-a068-3e0da37e4537">
Here you go.